packages/dart_frog_auth/lib/src/dart_frog_auth.dart

```diff
@@ -18,6 +18,12 @@ extension on Map<String, String> {
   String? basic() => authorization('Basic');
 }
 
+/// Function definition for the predicate function used by Dart Frog Auth
+/// middleware to determine if the request should be authenticated or not.
+typedef ApplyToRoute = Future<bool> Function(RequestContext);
```

Comment:
Since we rename the parameter to `applyToRoute` to `applies` due maybe we should consider renaming the typedef since it is also public?